main/webapp/modules/core/langs/translation-zh.json

```diff
@@ -490,7 +490,9 @@
         "log-length-facet": "文本长度的对数值归类",
         "unicode-facet": "Unicode字符归类",
         "facet-error": "按错误归类",
-        "facet-blank": "按空白归类",
+        "facet-blank": "按空白归类 (空值或空字符串)",
+        "facet-null": "按空值归类",
```

Comment:
I think maybe keep the null as-is is better to understand

Reply:
Can I double check what you mean by this?

Reply:
I mean from the translation aspect, it is not so clear to me what's the difference between those 3 facets.  按空值归类 is a little bit confusing even for the programmer, not mention the non-programmer end user. keeping null untranslated(as is) may not be best option, but it is better than confusion.

Reply:
The language fallback does not work so if you leave this string untranslated the button will show as blank in the UI.

Reply:
@jackyq2015 I can understand Chinese so this did not occur to me. @wetneb What do you make of @jackyq2015 's suggestion to use "按null归类" instead? That is fine with me if it helps users.

Reply:
@wetneb Not the whole string not get translated, just part of it(null). 
@joanneong I can tell you can understand Chinese from your name. I am from China but from the translation itself alone, it does not get cross the meaning of "facet by null". In a other word, there is no good Chinese translation for null. Instead of trying to translate it, I would rather keep the null as is. Like the "Unicode" new lines above

Reply:
The same thought had occurred to me with some other translations of `null` as it has quite a specific meaning in this context. I can't (sadly) comment on the specific of the translation, but I think it is OK (and sensible) to the term with specific meaning where there is no direct translation

Reply:
The Chinese version of Wikipedia does not try to translate it: https://zh.wikipedia.org/wiki/Null_(%E7%B7%A8%E7%A8%8B). It's interesting, I did not know that so many technical words were not translated at all.

Reply:
Ah I see! One more question, do you think we should just keep the word "null" across all the different languages, for consistency and in case such a scenario applies to other languages as well?

Reply:
On Wikipedia, it looks like null is never translated when it means `null` (and not nothing). It's a kind of constant, a bit like Pi.

Reply:
When I looked at this for #1500 I found that most translations used `null` but Spanish and Brazillian Portuguese translated it as "nulo". I copied this pattern when I updated the translation files for that PR on the basis that the people who did that translation were in a better position to make that call than me :)

I've been doing some looking round this morning and found the Microsoft terminology search on their language portal https://www.microsoft.com/en-us/language?sString=lock&langID=fr-fr. Based on this it seems that Microsoft translates in some languages and not others (and for Brazillian Portuguese it agrees with the use of `nulo`, while for Chinese it agrees with using `null`)

I suspect we could do worse than follow Microsoft practice here.

Note that the use of `null` in the transformation window as illustrated by @ettorerizza could equally be because this comes from the OR backend where we don't yet support any translations.

Reply:
But I also don't want to prolong this discussion - I'm happy with the decision either way - lets get this PR done! :)

Reply:
I can only speak for Chinese language. For this particular term, I believe keep it as-is is better. There might be good fit for null in other languages. But the principle we should follow is that if the translation cannot get across  the real meaning and we should NOT try to translate it to prevent further confusion.